Makefile

```diff
@@ -115,28 +117,28 @@ data/unicharset: $(ALL_BOXES)
 	unicharset_extractor --output_unicharset "$@" --norm_mode 1 "$(ALL_BOXES)"
 endif
 
-$(ALL_BOXES): $(sort $(patsubst %.tif,%.box,$(wildcard $(GROUND_TRUTH_DIR)/*.tif)))
+$(ALL_BOXES): $(patsubst %.tif,%.box,$(shell find $(GROUND_TRUTH_DIR) -name '*.tif'))
```

Comment:
Won't this reintroduce the issue https://github.com/OCR-D/ocrd-train/commit/bebc7db656aebd92ae3105873a2288724c438b99 tried to fix?

Reply:
I don't think so. It works fine with GT4HistOCR (more than 300000 data sets). There is no external shell command with a potentially too long argument list, only Makefile macros are used and these macros should work without arbitrary limits.

Reply:
In the old code the [lines 181 and 182](https://github.com/OCR-D/ocrd-train/commit/bebc7db656aebd92ae3105873a2288724c438b99#diff-b67911656ef5d18c4ae36cb6741b7965L181) would obviously fail with GT4HistOCR.